english/plow/plow.txt

```diff
@@ -0,0 +1,6 @@
+Mac: Great, onto the matter at hand. We're getting plowed in the ass by the oil companies and the gas companies with their ten gallon hats and their rotten ass-plowing hearts. So, as the brains of this organization, I came up with a plan.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.